train_management/templates/includes/tinymce.html

```diff
@@ -0,0 +1,28 @@
+<script>
+  tinymce.init({
```

Comment:
Regarding this, I thought it might be a good idea to create a generic tinymce object and init that in a asset file (e.g. `asssets/javascript/tinymce.js`). Then you could export that and reuse it for all tinymce-fields (e.g. using a specific class).
Also I think I understand what the templates are doing, but think that we should go with a different approach. Using these templates you will each time loading the edit view have to find check if there is already content in the db for this field and if yes load that and otherwise load the template with given data (since the content of this field is HTML as soon as you save once).
Instead you could add the HTML (as in your template) as the default value of the tinymce field. Therefore it would just update it whenever you edit that.